zos_concepts/templates/copy_template/zos_copy_template.yml

```diff
@@ -0,0 +1,167 @@
+###############################################################################
+# © Copyright IBM Corporation 2023
+###############################################################################
+
+###############################################################################
+# This playbook demonstrates how to use local Jinja templates to copy
+# datasets and files in z/OS using Red Hat Ansible Certified Content for IBM Z.
+#
+# Usage:
+#  ansible-playbook -i <inventory> <playbook>
+#
+# Example:
+#  ansible-playbook -i inventories zos_copy_template.yml
+#
+#  This playbook relies heavily on the files contained in the files
+#  directory, as these are the templates rendered by the tasks.
+#  Additional variables and Jinja blocks can be added to the files found
+#  in the files dir and this playbook. For more information about Jinja,
+#  see its official documentation at:
+#  https://jinja.palletsprojects.com/en/latest/templates/.
+#
+# Requirements:
+#   IBM z/OS core collection 1.7.0 or later.
```

Comment:
good idea to re-create that community.general package requirement text here too! This is in case someone grabs _just_ the playbook, they'll still have the requirement listed.